README.md

```diff
@@ -22,13 +22,17 @@ Ballerina also provides specially designed various database-specific DB connecto
 
 ### Client
 
-The database client should be created using any of the above-listed database libraries and once it is created, the operations and functionality explained below can be used.
+A database client can be created using any of the above-listed database libraries. The operations and functionality explained below can be used with the newly created client.
+
+> **Tip** : The client should be used throughout the application lifetime.
 
 #### Handle connection pools
 
 All database libraries share the same connection pooling concept and there are three possible scenarios for
 connection pool handling.  For its properties and possible values, see the [`sql:ConnectionPool`](https://docs.central.ballerina.io/ballerina/sql/latest/records/ConnectionPool).
 
+> **Tip** : Connection pooling is used to optimize opening and closing connections to the database. It can add up over time and waste precious network time. Therefore, it is best to open a limited connection pool and reuse connections according to the purpose rather than using default.
```

Comment:
Did you revert the changes here?

Reply:
https://github.com/ballerina-platform/module-ballerina-sql/pull/604#discussion_r1055535778

Reply:
This was fixed in [7a61380](https://github.com/ballerina-platform/module-ballerina-sql/pull/604/commits/7a61380413be5a99d5ad1577a4cfa114bf568d8b)

Reply:
Have you seen the latest changes, seems you have reverted back to the old sentences you used. This suggests connection pool will take up time and waste resources, when in reality it doesnt. Shall we go with the suggested changes, or are you planning on keeping this?

Reply:
Sorry, I haven't noticed you updated the message. I moved my one.